process-migration-service/frontend/src/component/tabMigrationPlan/wizardExecuteMigration/PageMigrationRunningInstances.js

```diff
@@ -266,7 +266,7 @@ export default class PageMigrationRunningInstances extends React.Component {
   };
   onRow = row => {
     const { selectedRows } = this.state;
-    const selected = selectedRows.indexOf(row.id) > -1;
+    const selected = selectedRows.indexOf(row.processInstanceId) > -1;
```

Comment:
const selected = selectedRows.indexOf(row.processInstanceId) > -1;
```suggestion
    const selected = selectedRows.includes(row.processInstanceId);
```